test/functional/viml/lang_spec.lua

```diff
@@ -8,7 +8,6 @@ describe('viml', function()
   it('parses `<SID>` with turkish locale', function()
     if exc_exec('lang ctype tr_TR.UTF-8') ~= 0 then
       pending("Locale tr_TR.UTF-8 not supported")
-      return
     end
```

Comment:
pending() does not return nor stop execution, does it?

Reply:
It does, see below.